packages/components/src/pages/VsCodeExtension.vue

```diff
@@ -193,6 +229,20 @@ export default {
       versionText: '',
       VIEW_COMPONENT,
       VIEW_FLOW,
+      filters: {
+        limitRootEvents: {
+          on: true,
+          default: true,
+        },
+        unlabeledEvents: {
+          on: true,
+          default: true,
+        },
+        hideMediaRequests: {
+          on: true,
+          default: true,
```

Comment:
What does the `default` key imply? Wouldn't the default be the initial value of `on`?

Reply:
`default` key is used in computed property `filtersChanged` to determine any changes in filters since initial load:
https://github.com/applandinc/appmap-js/blob/feat/global-filters/packages/components/src/pages/VsCodeExtension.vue#L393-L398

We need it to show small tip on filters icon when something was changed